### ISSUE TITLE
Compose tutorial update

### ISSUE THREAD

**Issue by abunur** [getditto/docs#628]
Compose tutorial needs update and housekeeping.
Rae, I assigned this to you so you can un-assign or assign back to me after determining scope / timeline for this update
PR for this issue needs to be separate from sample app update, different repo etc

I am currently updating the sample app that accompanies this tutorial, as it was not even compiling, and after fixing compile issues it doesn't work.
will update this issue as I come across things that need updating.
---
1. [Vector Icons Steps](https://docs.ditto.live/android/tutorial/jetpack-compose/setup) - remove this step. Instead, simply add the proper Compose library reference in Gradle; use [Material Icons](https://fonts.google.com/icons)
2. Update code snippets in tutorial to match updated Sample project
3. TBD

**Comment by okdistribute** [comment]
Yes sounds good -- first update the sample app, and then once you get that working the team can more easily copy/paste relevant code changes to the tutorial page.
